gateway-service/src/main/java/org/zowe/apiml/gateway/security/mapping/OIDCExternalMapper.java

```diff
@@ -29,6 +29,8 @@
 
 import java.io.UnsupportedEncodingException;
 
+import static org.zowe.apiml.gateway.security.mapping.model.MapperResponse.OIDC_FAILED_MESSAGE_KEY;
+
 @Slf4j
```

Comment:
is it possible to remove this logger?

Reply:
Actually no, I left there one more remaining `log.debug`.

Reply:
it can't be replaced with apimlLog?

Reply:
You are right, it can. I don't really mind using sl4j for debugging and tracing messages. But it is probably ugly to mix it in one class.